File: app/src/main/java/com/babestudios/companieshouse/data/model/officers/appointments/AppointedTo.java
package com.babestudios.companieshouse.data.model.officers.appointments;


import com.google.gson.annotations.SerializedName;

public class AppointedTo {

	@SerializedName("company_name")
	public String companyName;
	@SerializedName("company_number")
	public String companyNumber;
	@SerializedName("company_status")
	public String companyStatus;

}

File: app/src/main/java/com/babestudios/companyinfouk/data/model/officers/appointments/AppointedTo.kt
package com.babestudios.companyinfouk.data.model.officers.appointments


import com.google.gson.annotations.SerializedName

class AppointedTo {

	@SerializedName("company_name")
	var companyName: String? = null
	@SerializedName("company_number")
	var companyNumber: String? = null
	@SerializedName("company_status")
	var companyStatus: String? = null

}